run.sh

```diff
@@ -3,27 +3,35 @@ if [ ! -n "$WERCKER_NEWRELIC_DEPLOYMENT_API_KEY" ]; then
   exit 1
 fi
 
-if [ ! -n "$WERCKER_NEWRELIC_DEPLOYMENT_APP_NAME" ]; then
-  error 'Please specify app_name property'
+if [ ! -n "$WERCKER_NEWRELIC_DEPLOYMENT_APP_ID" ]; then
+  error 'Please specify app_id property'
   exit 1
 fi
 
 if [ ! -n "$WERCKER_NEWRELIC_DEPLOYMENT_REVISION" ]; then
-  export WERCKER_NEWRELIC_DEPLOYMENT_REVISION="$WERCKER_GIT_BRANCH/${WERCKER_GIT_COMMIT:0:10} by $WERCKER_STARTED_BY"
+  export WERCKER_NEWRELIC_DEPLOYMENT_REVISION="${WERCKER_GIT_BRANCH}/${WERCKER_GIT_COMMIT}"
```

Comment:
Lesser chance of conflicts. Since this is being passed to an API, and isn't just for display purposes, I think having the whole would be better.